Service/SourcesConverter.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Ampersand\DisableStockReservation\Service;
+
+use Magento\Framework\Serialize\SerializerInterface;
+use Magento\InventorySourceSelectionApi\Api\Data\SourceSelectionItemInterfaceFactory;
+
+/**
+ * Class SourcesConverter
+ * @package Ampersand\DisableStockReservation\Service
+ */
+class SourcesConverter
+{
+    /**
+     * @var SerializerInterface
+     */
+    private $serializer;
+
+    /**
+     * @var SourceSelectionItemInterfaceFactory
+     */
+    private $sourceSelectionItemInterfaceFactory;
+
+    /**
+     * SourcesConverter constructor.
+     * @param SerializerInterface $serializer
+     * @param SourceSelectionItemInterfaceFactory $sourceSelectionItemInterfaceFactory
+     */
+    public function __construct(
+        SerializerInterface $serializer,
+        SourceSelectionItemInterfaceFactory $sourceSelectionItemInterfaceFactory
+    ) {
+        $this->serializer = $serializer;
+        $this->sourceSelectionItemInterfaceFactory = $sourceSelectionItemInterfaceFactory;
+    }
+
+    /**
+     * @param array $sourcesItems
+     * @return string
+     */
+    public function convertSourceSelectionItemsToSourcesArray(array $sourcesItems): string
```

Comment:
This function returns a string, should it be called something like `convertSourceSelectionItemsToJson`?